sapi/tests/test004.phpt

```diff
@@ -6,10 +6,7 @@ if you use ScriptAlias configurations, and the proper
 path is in PATH_TRANSLATED.  SCRIPT_NAME in this is faked,
 but that is ok, Apache sets SCRIPT_NAME to the ScriptAlias
 of the executable.
---REQUEST--
-return <<<END
-PATH_INFO=/path/info
-END;
+--CGI--
```

Comment:
Based on https://github.com/php/php-src/blob/master/run-tests.php#L1555 it does not look like `--ENV--` is an evaluated section, so it should not be returning a doc string ... and at least in run-tests `$this->conf` is not a thing either, which is probably why the test fails.

Reply:
Ah, thanks! Indeed, this ENV section (as well as some others) only work by accident. I'm going to fix the wrong ENV sections which don't have unsupported sections, and the [docs](http://qa.php.net/phpt_details.php#env_section) in a separate commit.